pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -328,50 +364,105 @@ func DummyCmd() {
 	time.Sleep(200 * time.Millisecond)
 }
 
+var printOnce = true
+var diskDevice string // Based on largest disk
+var ddCount int       // Based on time for 200ms
+
+// InitDDCmd determines the disk (using the largest disk) and measures
+// the repetition count to get to 200ms dd time.
+func InitDDCmd(ledName string) {
```

Comment:
LOL @eriknordmark -- this is quite a dedication you're showing here with calibrating to 200ms and everything -- not sure that the functionality deserves this level of attention -- but it is fun to read